shoop/themes/classic_gray/plugins.py

```diff
@@ -26,18 +26,22 @@ class ProductHighlightPlugin(TemplatedPlugin):
             ("best_selling", "Best Selling"),
             ("random", "Random"),
         ], initial="newest")),
-        ("count", forms.IntegerField(label=_("Count"), min_value=1, initial=4))
+        ("count", forms.IntegerField(label=_("Count"), min_value=1, initial=4)),
+        ("orderable_only", forms.BooleanField(label=_("Only show in-stock and orderable items"),
+                                              initial=True,
+                                              required=False))
     ]
 
     def get_context_data(self, context):
         type = self.config.get("type", "newest")
         count = self.config.get("count", 4)
+        orderable_only = self.config.get("orderable_only", True)
```

Comment:
If the default value is True, this will change the behavior of shops currently using the plugin. Is that supposed to happen?